addons/spreadsheet/static/src/global_filters/plugins/global_filters_ui_plugin.js

```diff
@@ -229,17 +230,29 @@ export class GlobalFiltersUIPlugin extends spreadsheet.UIPlugin {
         const value = this.getGlobalFilterValue(filter.id);
         switch (filter.type) {
             case "text":
-                return value || "";
+                return { value: value || "" };
```

Comment:
maybe return an array every time ? woulad avoid one isArray()

Reply:
also the fucntion doesn't return a displayValue anymore, we should rename it

Reply:
> maybe return an array every time ? woulad avoid one isArray()

+1